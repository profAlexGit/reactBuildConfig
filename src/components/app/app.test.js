import React from 'react';
import { configure } from 'enzyme';
import ShallowRenderer from 'react-test-renderer/shallow';
import Adapter from 'enzyme-adapter-react-16';

import App from './app.jsx';
configure({ adapter: new Adapter() });

describe('<App />', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<App />);
    expect(tree).toMatchSnapshot();
  });
});
