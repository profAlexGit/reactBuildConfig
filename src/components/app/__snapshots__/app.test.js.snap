// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/auth"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/test"
  />
  <Route
    component={[Function]}
    exact={true}
    fetchData={[Function]}
    path="/"
  />
  <Route
    component={[Function]}
  />
</Switch>
`;
